### PR TITLE
Tests refactoring, removed deprecated code.

### PR DESCRIPTION
**Pull request by jeqq**

### CODE REVIEW

File: src/Tests/MultiversionManagerTest.php [hunk 1]
```diff
@@ -8,16 +8,22 @@
...
+use Drupal\entity_test\Entity\EntityTestRev;
+use Drupal\simpletest\WebTestBase;
 
 /**
  * Test the MultiversionManager class.
  *
  * @group multiversion
  */
-class MultiversionManagerTest extends MultiversionWebTestBase {
+class MultiversionManagerTest extends WebTestBase {
```
Comment (dickolsson) [inline src/Tests/MultiversionManagerTest.php]: Is there no way to make this use `KernelTestBase`? I believe you've tried, I just want to make sure :)

However, Inside `MultiversionManagerTest`, let's merge all tests into one single method... They are all so simple, so it doesn't make sense to re-install Drupal for each and every test here I think.

File: src/Tests/EntityStorageTest.php [hunk 2]
```diff
@@ -126,17 +128,14 @@ public function setUp() {
 
   public function testSaveAndLoad() {
```
Comment (dickolsson) [inline src/Tests/EntityStorageTest.php]: Maybe we can try to merge all test methods in here to one as well. We would save 4 installations of Drupal if we do that.

Maybe it's enough to clearly separate the save/update/delete/etc tests with inline comments?

File: src/Tests/MultiversionManagerTest.php (hunk 1 above)
Comment (jeqq) [inline src/Tests/MultiversionManagerTest.php]: @dickolsson We can't extend `KernelTestBase` for tests where we access our custom fields (in `MultiversionManagerTest` we use `_rev` and `_deleted`).

Comment (dickolsson) [inline src/Tests/MultiversionManagerTest.php]: @jeqq Ok then, I give up :wink:

**Pull request merged**